### DISCUSSION TITLE
Persistent detect status across reboots?

### DISCUSSION THREAD

**Discussion by FuzzyMistborn** [blakeblackshear/frigate#7060] [Ask A Question] [answered]
Been trying out Frigate and so far it's pretty great.  Probably will still use in conjunction with something like BlueIris until 24/7 recordings are added (I understand that's coming soon(tm)).

Question though: Is there a way to preserve during a reboot the "detect" setting being turned off?  As I've been playing around with my config, I've restarted a few times.  And I've noticed that when I turn detection off, it comes back on if I reboot.  Once I'm done tinkering obviously the reboots will be less, but still I'd expect if I turned detect off, it should stay off until I turn it back on.

**Comment by NickM-27** [comment, marked as answer]
> until 24/7 recordings are added (I understand that's coming soon(tm)).

frigate already supports 24/7 recordings and 24/7 recordings viewing

> Question though: Is there a way to preserve during a reboot the "detect" setting being turned off? As I've been playing around with my config, I've restarted a few times. And I've noticed that when I turn detection off, it comes back on if I reboot. Once I'm done tinkering obviously the reboots will be less, but still I'd expect if I turned detect off, it should stay off until I turn it back on.

No, that used to be supported but led to users being confused and also some bad states.

**Reply by FuzzyMistborn** [reply]
> frigate already supports 24/7 recordings and 24/7 recordings viewing

Thought I read that was coming in the next version?  Or maybe it was a better UI to view the recordings, because right now it's still largely even driven right?

> No, that used to be supported but led to users being confused and also some bad states.

Ahh that's too bad.  Guess I'll have to figure out a work around because I'd like some cameras to not detect most of the time and only detect at certain times.

**Reply by NickM-27** [reply]
> Thought I read that was coming in the next version? Or maybe it was a better UI to view the recordings, because right now it's still largely even driven right?

Not quite. In the next version there will be a UI to export recordings, and future improvements to the 24/7 recordings viewer are planned. But the 24/7 recordings viewer fully works now as it is along with the events viewer

> Ahh that's too bad. Guess I'll have to figure out a work around because I'd like some cameras to not detect most of the time and only detect at certain times.

it is not difficult to setup an automation in NodeRED or HomeAssistant to do this

**Reply by FuzzyMistborn** [reply]
> Not quite. In the next version there will be a UI to export recordings, and future improvements to the 24/7 recordings viewer are planned. But the 24/7 recordings viewer fully works now as it is along with the events viewer

Got it thanks!

> it is not difficult to setup an automation in NodeRED or HomeAssistant to do this

Can you give me a rough idea?  Right now that solution in my head would be an input_boolean or something to track whether I want detection on/off and then when the MQTT availability topic for Frigate changes to "available", run an automation to set the detection status to what I want?  Something simpler?

**Reply by NickM-27** [reply]
It could be done that way, I just have an automation that sets it to its default state with the assumption that I won't be restarting frigate at a time when I don't want that default state; and if I do I will change it manually

**Reply by FuzzyMistborn** [reply]
Much appreciated!  Thanks!
